cpp/ql/test/library-tests/dataflow/taint-tests/constructor_delegation.cpp

```diff
@@ -0,0 +1,47 @@
+
+int source();
+void sink(...);
+
+class MyValue
+{
+public:
+	MyValue(int _x) : x(_x) {}; // taint flows directly from _x to x
+	MyValue(int _x, bool ex) : MyValue(_x) {}; // taint flows indirectly from _x to x
+	MyValue(int _x, int _y) : MyValue(_x + _y) {}; // taint flows indirectly from _x to x
```

Comment:
In the comments here, does `_x` refer to the parameter declaration or to the access in the init list?

Reply:
Yep, these comments weren't very clear.  I've just push an improvement.